etna/pipeline/base.py

```diff
@@ -88,7 +88,7 @@ def validate_on_dataset(self, ts: TSDataset, horizon: int):
         dataset_timestamps = list(ts.index)
         dataset_description = ts.describe()
 
-        min_first_timestamp = dataset_description["start_timestamp"].min()
+        min_first_timestamp = ts.index.min()
```

Comment:
I guess we need to fix the first_train_timestamp itself [here](https://github.com/tinkoff-ai/etna/blob/f8e74c183d4d6e52de4b72db149036e916a1f8e5/etna/pipeline/base.py#L467), it should be not the first dataset index, but the first valid index